docsrc/imap/concepts/deployment/databases.rst

```diff
@@ -223,7 +223,7 @@ have that conversations ID in them, and counts of messages and flags.
 Finally there are records for each folder with the counts of conversations in
 that folder.
 
-File type can be: `skiplist`_ (default), `sql`_, or `twoskip`_.
+File type can be: `skiplist`_ (default), `sql`_, `twoskip`_, or `twom`_.
```

Comment:
imapoptions says the default is `twom`.

Reply:
hmm, I should undo any default changes I think; we should only change defaults in major versions.

Reply:
Yeah, fixed the default back - as bad as skiplist is as a choice for conversationsdb, it's the default for the current versions